src/domain/youthProfile/create/CreateYouthProfile.tsx

```diff
@@ -51,12 +57,18 @@ function CreateYouthProfile({
   const { i18n } = useTranslation();
   const { trackEvent } = useMatomo();
   const history = useHistory();
+  const profileApiToken = useSelector(profileApiTokenSelector);
 
-  const [createProfile, { loading }] = useMutation<
+  const [createProfile, { loading: creatingProfile }] = useMutation<
     CreateMyProfileData,
     CreateMyProfileVariables
   >(CREATE_PROFILE);
 
+  const [createMyYouthProfile, { loading: creatingYouthProfile }] = useMutation<
```

Comment:
This approach works and is fine, but I'd consider whether it'd make sense to encapsulate it more. You could for instance have a custom hook which hides the two mutations calls behind a single call. This is a flaky spot in the architecture of this application in some sense, because we are integrating two APIs and encapsulation would provide some level of protection against regressions.

Reply:
Good idea, I'll create a custom hook for this 👍